scripts/cibuild.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+vagrant up --provision
+vagrant ssh -c "
+    cd /vagrant
+    echo -e '\nRunning linter...'
+
+    # run linter with console output
+    npm run lint
+
+    # run linter with file output for Jenkins
+    npm run lint:ci
```

Comment:
This script is only intended to be run in the Jenkins environment (this is the `cibuild` script, only for building on the CI server.)